src/test/groovy/edu/ie3/datamodel/utils/validation/SystemParticipantValidationUtilsTest.groovy

```diff
@@ -294,8 +294,9 @@ class SystemParticipantValidationUtilsTest extends Specification {
     ex.message.contains(expectedException.message)
 
     where:
-    invalidHpType                                                                                           || expectedException
-    new HpTypeInput(uuid, id, capex, opex, sRated, cosPhiRated, Quantities.getQuantity(0, ACTIVE_POWER_IN)) || new InvalidEntityException("The following quantities have to be positive: 0 kW", invalidHpType)
+    invalidHpType                                                                                             || expectedException
+    new HpTypeInput(uuid, id, capex, opex, Quantities.getQuantity(0, S_RATED), cosPhiRated, pThermal) || new InvalidEntityException("The following quantities have to be positive: 0 kVA", invalidHpType)
```

Comment:
Wouldn't it meke more sense to write the name of the field here?

Reply:
Yes, but I would opt to do this in a new issue. Since its some quite generic method that is used multiple times so I guess there will be more refactoring then.